Sources/DistributedActors/utils.swift

```diff
@@ -99,6 +110,17 @@ func pprint(_ message: StaticString, file: String = #filePath, line: UInt = #lin
     """)
 }
 
+func pprint(_ message: StaticString, metadata: [String: Any], file: String = #filePath, line: UInt = #line) {
+    print("""
+    [pprint]\
+    [\(_createTimeFormatter().string(from: Date()))] \
+    [\(file):\(line)]\
+    {\(metadata.map { "\($0):\($1)" }.joined(separator: ";"))}
+    [\(_hackyPThreadThreadId())]: \
+    \(message)
+    """)
+}
```

Comment:
not used in commit; but I use those println debugging a lot when debugging issues, so adding them